styles.css

```diff
@@ -25,36 +25,505 @@
 /* ------------------------------- 1.1. Colors ----------------------------------- */
 
 :root {
-    --dark-primary: #5722CD;
+    --dark-primary: hsl(259, 72%, 47%);
+    --dark-primary-hover: hsl(259, 72%, 80%);
     --blue-primary: #4845E4;
-    --pink-primary: #ED4385;
+    --pink-primary: hsl(337, 83%, 60%);
+    --pink-primary-hover: hsl(337, 83%, 70%);
     --black-copy: #0C0C0C;
     --purple-secondary: #F2F0FF;
     --light-grey: #F3F6F9;
     --regular-grey: #7A7A7A;
+    --container-small: 50%;
+    --container-medium: 60%;
+    --container-large: 80%;
+    --gradient: linear-gradient(90deg, rgba(72, 69, 228, 1) 0%, rgba(87, 34, 205, 1) 73%);
+    --navbar-height: 3rem;
+    --box-shadow-btn: 0 5px 5px 2px rgba(0, 0, 0, 0.10);
+    --box-shadow-image: 0px 6px 12px 2px rgba(0, 0, 0, 0.10);
 }
 
+
 /* ------------------------------- 2. Typography ----------------------------------- */
 
-p, a, li, span {
+body {
     font-family: "Roboto", sans-serif;
 }
 
 h1, h2, h3, h4, h5, h6 {
     font-family: "Merriweather", sans-serif;
 }
 
+input, textarea, button {
+    font-family: inherit;
+}
+
+h1 {
+    font-size: 2.8rem;
+    color: var(--light-grey);
+}
+
+h2 {
+    font-size: 1.6rem;
+    color: var(--black-copy);
+}
+
+h3 {
+    font-size: 1.3rem;
+    color: var(--dark-primary);
+}
+
 
 /* ------------------------------- 3. Layout --------------------------------------- */
 
+body {
+    background-color: #fff;
+}
+
+
 /* --------------- 3.1 Reusables ----------------------- */
 
+/* Op de inner/outer containers zo min mogelijk styling zetten om de layout functionaliteit puur te houden.*/
+.outer-container {
```

Comment:
Het principe van inner en outer container is volledig correct, maar zelf ben ik meer fan van één enkele container met de volgende declaraties:

```css
.container {
  max-width: 1200px;  /* Maximale breedte van de inhoud */
  margin: 0 auto;     /* Centreert de container horizontaal */
  padding: 0 1rem;    /* Witruimte links en rechts (responsive) */
}
```
Wat uiteindelijk hetzelfde effect kan opleveren als het gebruik van een inner en een outer container. Het voordeel is dat je minder utility classes hebt en minder onnodige diffs hoeft toe te voegen om je layout te kunnen maken. 

Dus deze comment kun je zien als een nice to know, want je hebt het zeker goed toegepast.